src/lib/y2storage/proposal/autoinst_devices_creator.rb

```diff
@@ -358,6 +363,11 @@ def set_up_lvm(vgs, devs_to_reuse)
         vgs.reduce(creator_result) do |result, vg|
           pvs = creator_result.created_names { |d| d.pv_for?(vg.volume_group_name) }
           pvs += devs_to_reuse.select { |d| d.pv_for?(vg.volume_group_name) }.map(&:reuse_name)
+          if pvs.empty?
+            issues_list.add(:no_pvs, vg)
+            next result
```

Comment:
NP: always when I see this reduce and next handling I am a bit scared that it is forgotten. What about using instead each_with_object that is safer for next and break, as it does not need to have parameter?

Reply:
For the time being, I trust in the test suite to catch that problem. But I will have it into account in the future.